src/map/battle.c

```diff
@@ -2323,9 +2323,9 @@ int battle_calc_skillratio(int attack_type, struct block_list *src, struct block
 					break;
 				case NC_ARMSCANNON:
 					switch( tst->size ) {
-						case SZ_SMALL: skillratio = 300 + 350 * skill_lv; break; // Medium
-						case SZ_MEDIUM: skillratio = 300 + 400 * skill_lv; break;  // Small
-						case SZ_BIG: skillratio = 300 + 300 * skill_lv; break;    // Large
+						case MOBSIZE_SMALL: skillratio = 300 + 400 * skill_lv; break;  // Small
+						case MOBSIZE_MEDIUM: skillratio = 300 + 350 * skill_lv; break; // Medium
+						case MOBSIZE_BIG: skillratio = 300 + 300 * skill_lv; break;    // Large
```

Comment:
according to this site
http://en.kafra.ru/skills/detail/2261-nc_armscannon.html
the bigger the monster, deal more damage, which means this was bug all the time
or my info was wrong